pangram3.py

```diff
@@ -0,0 +1,43 @@
+# Solution 3 - O(n^2) Quadratic Time
+
+# Solution where a complete alphabet array is not needed to check against
+
+# I *think* that the alphabet array would change the amount of time it takes for the computer to solve the problem, based on the order of alphabets. So tried to take that array out of the equation.
+
+import time
+
+def checkIfPangramSolutionThree(n):
+  start = time.time()
+
+  # Make sure it's a string
+  if isinstance(n, str) != True:
+    return False
+
+  # Clean up the string of spaces and turn it into an array, sorted A-Z
+  ## O(1) Constant operations
+  removeSpaces = n.replace(' ','')
+  toLowerCase = removeSpaces.lower()
+  ## toLowerCase runs a For loop + if statement so O^n2 Quadratic time
+  createList = list(toLowerCase)
+  ## Sorted list O(nlogn) logarithmic time
+  sortedList = sorted(createList)
+
+  # Remove duplicates
+  ## Set appears to be an O(1) Constant operation
```

Comment:
Turning a list into a set is actually linear, since duplicates need to be removed (and the data needs to be copied to a different place in memory, formatted differently).